templates/2_create_tiled_mibi_run.ipynb

```diff
@@ -13,12 +13,14 @@
    "cell_type": "markdown",
```

Comment:
<p>I think this one got condensed a bit too much. When defining variables, I think the format from the previous version is good for users, since it visually separates each variable. </p><p> </p><p>Adding a visual layout for what the directory will look like is good idea, but if it's going to describe the directory structure, it should do just that, not also try and explain each file. It's also hard to tell that this is actual a visual layout of a directory structure, I would copy the format Sri used for the Mantis directory structure in the ark readme</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Sounds good, I'm going to copy over the same format we have from <code>2_create_tma_mibi_run.ipynb</code> where we put the prefix up top, then have all the file definitions in one separate cell.</p><p> </p><p>Documentation for this section will also be organized like that section.</p>

Reply:
<p>Now that I think about it, a visual directory structure image might be a bit much to add considering there are only a few files in one root directory, with no underlying subdirectory structure.</p><p> </p><p>The TMA notebook seems intuitive enough without one. Do you think this notebook can do without it as well, or should we keep it?</p>

Reply:
Up to you, whatever you think works best